admin/class-admin-apple-json.php

```diff
@@ -180,6 +180,15 @@ public function page_json_render() {
 		// Negotiate selected theme.
 		$selected_theme = $this->get_selected_theme();
 
+		// Extract theme layout configuration.
+		$layout_columns = '';
+		$layout_width   = '';
```

Comment:
Nitpicky, but these are integers, so I'd recommend setting the default value to `0`.